### DISCUSSION TITLE
Tanya

### DISCUSSION THREAD

**Discussion by snjiwan** [dewanakl/undangan#26] [General]
Mass ini dashboardnya emng gabisa dibuka yaa ? 
kalau di buka gmna caranya

**Comment by dewanakl** [comment]
Bisa mas, step yang pertama pastikan sudah deploy sendiri untuk api nya mas

**Comment by snjiwan** [comment]
berarti harus deploy sendiri dulu baru bisa mas ?


Pada Sab, 25 Mei 2024 pukul 19.24 Dewana Kretarta Lokeswara <
***@***.***> menulis:

**Reply by dewanakl** [reply]
Betul mas

**Reply by snjiwan** [reply]
saya masi bingung mas untuk deploy api nya tuh gmna yaa agar masuk ke saya

**Reply by dewanakl** [reply]
Apakah ada kendala error?
Untuk step stepnya ada di readme ini ya mas

**Reply by snjiwan** [reply]
trus habis ke readme gmna mas ?

**Reply by dewanakl** [reply]
Bisa lihat video untuk deployment api nya mas

**Comment by snjiwan** [comment]
buat cara connect htmlnya ke db gmna mas ?

**Comment by snjiwan** [comment]
di bagian mana yah mas connctnya ke html dari db

Pada Kam, 30 Mei 2024 08.16, Dewana Kretarta Lokeswara <
***@***.***> menulis:

**Reply by dewanakl** [reply]
Melalui api nya kak
ada pada slide presentasinya
